docs/cmdline-opts/tls-sessiontickets.d

```diff
@@ -0,0 +1,7 @@
+Long: tls-use-session-tickets
+Arg:
+Help: Enable TLS session tickets (NSS)
+Added: 7.6x.0
+---
+Set TLS Session Tickets in NSS for Session Resumption without Server-Side
+State (RFC 5077)
```

Comment:
I think this explanation should expand a little on why a user would like to use this option even if they're not experts in TLS protocol details.

Perhaps drop "use" part from the flag name to shorten it somewhat?

Reply:
done